src/components/layout/integrations/recipes/RecipeItemDetail.js

```diff
@@ -135,11 +104,11 @@ export const RecipeItemDetail = ({
       <RecipeInfo>
         <IntegrationImage icon={icon} accent={accentColor} withConnector />
         <div>
-          <Title isLoading={isLoading}>
-            <span>{isLoading ? 'loading' : `Integrate ${formattedName} and Storybook`}</span>
+          <Title>
+            <span>{`Integrate ${formattedName} and Storybook`}</span>
```

Comment:
Super minor, but I don't think we need the inner `span`s anymore?

Reply:
You're making a whole lot of sense, Mr. Gach!